modules/elements-color-picker/assets/js/editor/commands/apply.js

```diff
@@ -0,0 +1,9 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Apply extends CommandBase {
+	apply( args ) {
+		// Silent
+		this.component.currentPicker.container.settings.set( this.component.currentPicker.control, args.value );
```

Comment:
Before changing the value, save it for `ExitPreview`.
`this.component.currentPicker.prevValue`